components/kyma-environment-broker/common/orchestration/strategies/parallel.go

```diff
@@ -221,8 +220,9 @@ func (p *ParallelOrchestrationStrategy) updateMaintenanceWindow(execID string, o
 		}
 
 		duration = time.Until(op.MaintenanceWindowBegin)
-
-	case orchestration.Immediate:
+	} else {
+		p.executor.Reschedule(id, strategy.ScheduleTime, strategy.ScheduleTime)
+		duration = time.Until(strategy.ScheduleTime)
```

Comment:
It does work well when `strategy.ScheduleTime.IsZero()`, right? So the duration is <= 0 in that case (could be checked on https://go.dev/play/)